packages/navbrand/src/react/__specs__/index.spec.tsx

```diff
@@ -1,20 +1,51 @@
 import { render } from '@testing-library/react'
 import React from 'react'
 
-import Navbrand from '../index.js'
+import NavBrand from '..'
 
-describe('Navbrand', () => {
-  it('renders', () => {
-    const { getByTestId } = render(<Navbrand data-testid="undertest" />)
+describe('NavBrand', () => {
+  it('should render a div by default', () => {
+    const ref = React.createRef<HTMLDivElement>()
+    const { getByTestId } = render(
+      <NavBrand data-testid="undertest" ref={ref}>
+        test
+      </NavBrand>
+    )
+    const el = getByTestId('undertest')
 
-    expect(getByTestId('undertest')).toBeInTheDocument()
+    expect(el.tagName.toLowerCase()).toEqual('div')
+    expect(ref.current).not.toBeNull()
   })
 
-  it('forwards refs', () => {
-    const ref = React.createRef()
+  it('should render a hyperlink when an `href` is present', () => {
+    const ref = React.createRef<HTMLAnchorElement>()
+    const { getByTestId } = render(
+      <NavBrand data-testid="undertest" href="/" ref={ref}>
+        test
+      </NavBrand>
+    )
+    const el = getByTestId('undertest')
+
+    expect(el.tagName.toLowerCase()).toEqual('a')
```

Comment:
Why not rely on the stories that seem to test for the renderings of `<a>`s and `<button>`s?

Reply:
mostly just for TDD purposes. I like that they are focused and the test name is explicit about what is tested